src/OctoshiftCLI.Tests/gei/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -939,7 +951,10 @@ public async Task It_Uses_Github_Source_Pat_When_Provided()
             _mockSourceGithubApi.Setup(x => x.GetArchiveMigrationStatus(SOURCE_ORG, It.IsAny<int>()).Result).Returns(ArchiveMigrationStatus.Exported);
             _mockSourceGithubApi.Setup(x => x.GetArchiveMigrationStatus(SOURCE_ORG, It.IsAny<int>()).Result).Returns(ArchiveMigrationStatus.Exported);
 
-            _mockAzureApi.Setup(m => m.UploadToBlob(It.IsAny<string>(), It.IsAny<byte[]>()).Result).Returns(new Uri("https://example.com/resource"));
+            _mockHttpDownloadService.Setup(x => x.DownloadToFile(It.IsAny<string>(), It.IsAny<string>()));
+            _mockFileSystemProvider.Setup(x => x.OpenRead(It.IsAny<string>())).Returns(It.IsAny<FileStream>);
```

Comment:
Same here too, you'll get these setups for free.

Reply:
Addressed in 7d436e04071bf868990e1d32d6276668c1db1a35 :tada: